### PR TITLE
Release 2.4.7

### PR DESCRIPTION
**Pull request by johanandren**

### CODE REVIEW

**Comment by ktoso** [comment]
LGTM

File: docs/index.md [hunk 1]
```diff
@@ -16,20 +16,20 @@ title: Akka Documentation
       <option>Java</option>
     </select>
   </li>
-  <li><a href="http://doc.akka.io/docs/akka/2.4.6">reference documentation</a>.</li>
+  <li><a href="http://doc.akka.io/docs/akka/2.4.7">reference documentation</a>.</li>
```
Comment (patriknw) [inline docs/index.md]: What do you guys think about having a symlink `2.4.x` -> `2.4.7` (latest patch release)?
are there any disadvantages of that? we would still release each one as today, but the entry links from akka.io would link to `2.4.x`. Perhaps helps with google ranking also?

Not for today, but something to think about.

Comment (ktoso) [inline docs/index.md]: Good idea, I'd +1 that

Comment (johanandren) [inline docs/index.md]: Anything making us not having to replace a string in four different text files sounds good to me...

**Pull request merged**